app/services/media_storage/stored_path.rb

```diff
@@ -10,11 +10,14 @@ def self.media_url(url, stored_path)
       PublicSuffix.parse(uri.host)
       # we have a valid domain prefix here so remove it
       # to allow us to construct the URL correctly
+      env_prefix = '/' + Rails.env
```

Comment:
I assume we won't ever have a `stored_path` value of `production/user_avatar/1e5fc9b5-86f1-4df3-986f-549f02f969a5.jpeg` (no leading `/`) ?

If we do this env_prefix won't strip the value correctly due to missing the `/`

Reply:
The value we're checking in this case will always be the `.path` value of the URI object. `.path` will reliably return the path with a leading `/`, so we will never end up with a value without a leading `/`.